assets/multiqc_config.yml

```diff
@@ -12,6 +12,7 @@ report_section_order:
 
 export_plots: true
 
+disable_version_detection: true
```

Comment:
what's happening there?

Reply:
That's from template, I tihnk it turns off the multiqc scanning of versions from logs as we have it ourself in the pipelines